src/main/java/utc/k61/cntt2/class_management/service/ExamScoreService.java

```diff
@@ -123,6 +124,27 @@ public Page<?> fetchAllExam(Long classId) {
         return new PageImpl<>(classroom.getExams());
     }
 
+
+    public Object getExamScore(Long classRegistrationId){
+        User currentLoginUser = userService.getCurrentUserLogin();
+        if (currentLoginUser.getRole().getName() != RoleName.TEACHER) {
+            throw new BusinessException("Require Role Student!");
```

Comment:
Why conditional is != RoleName.TEACHER that message is Require Role Student!?

Reply:
because I think teacher mangage the information of students, and teacher see the score of student , so, essentially check currentUserLogin is TEACHER , then they can continue...